CHANGES.rst

```diff
@@ -1,17 +1,21 @@
 ``zope.event`` Changelog
 ========================
 
-4.1.1 (unreleased)
+4.2.0 (unreleased)
 ------------------
 
-- Claim support for Python 3.5.
+- Add support for Python 3.5.
```

Comment:
I'm not a native English speaker ;)

I haven't seen this distinction noted anywhere, and I don't really care about it myself.  Now that I think about it more, "Add Python 3.x support" can be interpreted "we promise Python 3.x support starting with this version", which is true and independent of the changes you needed or didn't need to make to the codebase to make it work.